packages/calling-stateful-client/review/beta/calling-stateful-client.api.md

```diff
@@ -291,7 +301,9 @@ export type ReactionState = {
 
 // @public
 export interface RecordingCallFeature {
+    activeRecordings?: RecordingInfo[];
```

Comment:
This recording call feature is different than local recording? If we are focused on local recording notification, why do we need to change this feature's obj?

Reply:
I mentioned in our group chat, but we also need to change the recording feature as there is a requirement to also retrieve the displayname for the person that started 'cloud' recording. 

Cloud recording = RecordingCallFeature
Local recording = LocalRecordingCallFeature